Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -70,10 +310,18 @@ def main():
         client.set_credentials(api_user, api_key)
 
         if command == 'test-module':
-            result = test_module(client)
+            return_results(test_module(client))
         else:
-            raise NotImplementedError(f'Command {command} is not implemented')
-        return_results(result)
+            ILLUMIO_COMMANDS = {
```

Comment:
This isn't a constant, don't use capital_case here.

Reply:
Yes